### PR TITLE
Fix error when you click in back button

### PR DESCRIPTION
**Pull request by wilrona**
- Fix : https://new-legacy-digital-ltd.sentry.io/issues/4703640645
- Fix : https://new-legacy-digital-ltd.sentry.io/issues/4684453308/

### CODE REVIEW

File: packages/client/src/views/PrintCertificate/collectorForm/CollectorForm.tsx [hunk 1]
```diff
@@ -24,6 +24,7 @@ import {
 } from '@client/declarations'
 import { FormFieldGenerator } from '@client/components/form'
 import {
+  CertificateSection,
```
Comment (rikukissa) [inline packages/client/src/views/PrintCertificate/collectorForm/CollectorForm.tsx:27]: This us unused, right?

File: packages/components/src/Workqueue/Workqueue.tsx [hunk 2]
```diff
@@ -190,32 +190,34 @@ export class WorkqueueComp extends React.Component<
...
-                      isSorted={Boolean(preference.isSorted)}
-                      isDescending={sortOrder === SORT_ORDER.DESCENDING}
-                    />
-                  )}
-                </ColumnTitleWrapper>
-              </ColumnContainer>
-            ))}
-          </TableHeader>
-        )}
+        {content.length > 0 &&
```
Comment (rikukissa) [inline packages/components/src/Workqueue/Workqueue.tsx:193]: Is this just a formatting change or can you point me to what changed?

Comment (wilrona) [reply packages/components/src/Workqueue/Workqueue.tsx:193]: This is simply a change of format.

File: packages/client/src/views/PrintCertificate/collectorForm/CollectorForm.tsx (hunk 1 above)
Comment (wilrona) [reply packages/client/src/views/PrintCertificate/collectorForm/CollectorForm.tsx:27]: yes

File: packages/client/src/views/PrintCertificate/VerifyCollector.tsx [hunk 3]
```diff
@@ -252,7 +252,16 @@ const mapStateToProps = (
...
+
+  let registerForm
+
+  if (declaration) {
+    registerForm = getEventRegisterForm(state, declaration.event)
+  } else {
+    registerForm = {
+      sections: []
+    }
+  }
```
[the remark this answers is not in the document]
Comment (wilrona) [reply packages/client/src/views/PrintCertificate/VerifyCollector.tsx:264]: done

[the remark this answers is not in the document]
Comment (wilrona) [reply packages/client/src/views/PrintCertificate/VerifyCollector.tsx:264]: The approach can be approved, but it requires corrections to be made to remove typescript errors. Take a closer look at my modifications to the file and give me your feedback

**Review by rikukissa — CHANGES_REQUESTED** [review]

File: packages/client/src/views/PrintCertificate/VerifyCollector.tsx [hunk 4]
```diff
@@ -75,13 +75,18 @@ type IFullProps = IStateProps & IDispatchProps & IOwnProps
...
-    const eventDate = getEventDate(this.props.declaration.data, event)
-    const registeredDate = getRegisteredDate(this.props.declaration.data)
+
+    if (!this.props.declaration) {
+      return
+    }
+
+    const event = this.props.declaration?.event
+    const eventDate = getEventDate(this.props.declaration?.data, event)
+    const registeredDate = getRegisteredDate(this.props.declaration?.data)
```
[the remark this answers is not in the document]
Comment (wilrona) [reply packages/client/src/views/PrintCertificate/VerifyCollector.tsx]: Alright

**Review by rikukissa — APPROVED** [review]

**Pull request merged**